android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainActivity.kt

```diff
@@ -300,4 +304,36 @@ open class AppMainActivity : BaseMultiLanguageActivity(), QuestionnaireHandler,
       }
     }
   }
+
+  private fun overrideOnBackPressListener() {
+    onBackPressedDispatcher.addCallback(
+      object : OnBackPressedCallback(true) {
+        override fun handleOnBackPressed() {
+          val navController =
+            (supportFragmentManager.findFragmentById(R.id.nav_host) as NavHostFragment)
+              .navController
+          if (
+            navController.currentDestination?.id?.equals(R.id.geoWidgetLauncherFragment)!! ||
+              navController.currentDestination?.id?.equals(R.id.registerFragment)!!
+          ) {
+            val title =
+              appMainViewModel.applicationConfiguration.appExitDialog?.title
```

Comment:
While it doesn't hurt to have it I don't think we need to make this configurable from App Configs. This is because the strings are already configurable via the **strings.xml** file and such a message is mostly likely only going to be configured when releasing an app and rarely after that. And since all apps should have it, it should not be optional (which would have justified the config)

I think we want to leave the app config level configurability for settings that can change frequently and most likely impact workflows which change regularly cc @ellykits

Reply:
@ellykits so what's the consensus on this? SHould I go ahead and remove the app config part?

Reply:
I missed that we can override the `strings` in the build variant. So yes, please we do not need the app configs now.

Reply:
Updated